ocs_ci/helpers/helpers.py

```diff
@@ -4219,3 +4219,25 @@ def disable_vm_network_for_duration(
     logger.info(f"Enabled network connectivity for virtual machine {ip}")
 
     return True
+
+
+def verify_only_ocs_nodes_in_storagecluster():
```

Comment:
can we rename it to, verify_storagecluster_nodetopology()

Reply:
done